tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -114,6 +117,15 @@ def test_params_default(self):
                     self.Parameter(name='test', desc='  test description2 \n')]
         self.check_docstring(doc, expected)
 
+    def test_excps_default(self):
```

Comment:
@jayvdb This is the intended behaviour. Right now extra white spaces are preserved. As this should be later handled by bears(analysing routines). Anyway, removed those inappropriate whitespace and I m amending  a new test case with improper whitespace in `default.py` :+1: